eksconfig/README.md

```diff
@@ -36,6 +36,7 @@ AWS_K8S_TESTER_EKS_ADD_ON_STRESSER_REMOTE_ENABLE=true \
 AWS_K8S_TESTER_EKS_ADD_ON_CLUSTER_LOADER_LOCAL_ENABLE=true \
 AWS_K8S_TESTER_EKS_ADD_ON_CLUSTER_LOADER_REMOTE_ENABLE=true \
 AWS_K8S_TESTER_EKS_ADD_ON_CLUSTER_VERSION_UPGRADE_ENABLE=true \
+AWS_K8S_TESTER_ADD_ON_CLUSTER_AUTOSCALER_ENABLE=true \
```

Comment:
We do not want to cluster-wide autoscaler enable. This should be scoped down to individual node groups. I think your code already does that. Can we remove this?

Reply:
> We do not want to cluster-wide autoscaler enable. This should be scoped down to individual node groups. I think your code already does that. Can we remove this?

forgot t remove it.